src/custom/components/Popups/TransactionPopupMod.tsx

```diff
@@ -1,13 +1,11 @@
-import React, { useContext } from 'react'
+import React, { useContext, useMemo } from 'react'
 import { AlertCircle, CheckCircle } from 'react-feather'
 import styled, { ThemeContext } from 'styled-components'
 import { useActiveWeb3React } from 'hooks'
-import { TYPE } from 'theme'
-// import { ExternalLink } from 'theme'
-// import { getEtherscanLink } from 'utils'
+import { TYPE, ExternalLink } from 'theme'
 import { AutoColumn } from 'components/Column'
 import { AutoRow } from 'components/Row'
-import { ExplorerLink } from '../ExplorerLink'
+import { getEtherscanLink, getExplorerLabel } from 'utils'
```

Comment:
I think the solution should be to make `ExplorerLink` to be agnostic of the underlying explorer we use. 

That was certainly the intention. I'll try to make an alternative proposal

Reply:
it is pointing at #247  now - that was my intention (hence the description)